code/modules/antagonists/gang/gang_member.dm

```diff
@@ -0,0 +1,75 @@
+/datum/antagonist/subordinate/gang_member
+	id = ROLE_GANG_MEMBER
+	display_name = "gang member"
+
+	/// The gang that this gang member belongs to.
+	var/datum/gang/gang
+	/// The headset of this gang member, tracked so that additional channels may be later removed.
+	var/obj/item/device/radio/headset/headset
+
+	New(datum/mind/new_owner, do_equip, do_objectives, do_relocate, silent, source, do_pseudo, do_vr, late_setup, master)
+		src.master = master
+		src.gang = src.master.current.get_gang()
+		src.gang.members += new_owner
+
+		. = ..()
+
+		src.gang.leader.current?.antagonist_overlay_refresh(TRUE, FALSE)
+		for(var/datum/mind/M in src.gang.members)
```

Comment:
is the leader not a member?

Reply:
Gang leaders are not added the the `members` list, no. I'll code document `/datum/gang` to clarify this.